hamming.js

```diff
@@ -19,14 +19,13 @@ Hamming.prototype.compute = function (strand1, strand2) {
                 dif.push(i);
 
             }           
-
+  return(dif);
```

Comment:
Maybe you want to `return` the `dif.length`, and do it at the same line than the `console.log` :wink: !